### ISSUE TITLE
Large kmem_alloc with 0.6.5.7

### ISSUE THREAD

**Issue by woffs** [zfsonlinux/zfs#6099]
### System information
Type                                | Version/Name
  ---                                  |     --- 
Distribution Name       | debian
Distribution Version    | wheezy
Linux Kernel                 | 3.2.88
Architecture                 | amd64
ZFS Version                  | 0.6.5.7
SPL Version                  | 0.6.5.7

### Describe the problem you're observing

kernel message "Large kmem_alloc"

### Describe how to reproduce the problem

zpool online -e from 50GB to 800GB

### Include any warning/errors/backtraces from the system logs
```
[606414.274757] Large kmem_alloc(51192, 0x1000), please file an issue at:
[606414.274760] https://github.com/zfsonlinux/zfs/issues/new
[606414.274973] Pid: 19526, comm: spa_async Tainted: P           O 3.2.0-4-amd64 #1 Debian 3.2.88-1
[606414.275133] Call Trace:
[606414.275209]  [<ffffffffa0250771>] ? spl_kmem_alloc_impl+0x64/0x111 [spl]
[606414.275365]  [<ffffffffa0353b84>] ? vdev_metaslab_init+0x86/0x1a9 [zfs]
[606414.275525]  [<ffffffff81353835>] ? mutex_lock+0xd/0x2d
[606414.275653]  [<ffffffffa034c493>] ? spa_config_enter+0xdc/0xfe [zfs]
[606414.275790]  [<ffffffffa0356b16>] ? vdev_expand+0x1f/0x56 [zfs]
[606414.275922]  [<ffffffffa034a696>] ? spa_config_update+0x81/0xe0 [zfs]
[606414.276056]  [<ffffffffa03490ca>] ? spa_async_thread+0xa3/0x292 [zfs]
[606414.276168]  [<ffffffffa02524d5>] ? thread_generic_wrapper+0x60/0x75 [spl]
[606414.276305]  [<ffffffffa0349027>] ? spa_vdev_resilver_done+0xef/0xef [zfs]
[606414.276426]  [<ffffffffa02524df>] ? thread_generic_wrapper+0x6a/0x75 [spl]
[606414.276542]  [<ffffffffa0252475>] ? __thread_create+0x11c/0x11c [spl]
[606414.276666]  [<ffffffff8105fbf1>] ? kthread+0x76/0x7e
[606414.276764]  [<ffffffff8135b5f4>] ? kernel_thread_helper+0x4/0x10
[606414.276868]  [<ffffffff8105fb7b>] ? kthread_worker_fn+0x139/0x139
[606414.276972]  [<ffffffff8135b5f0>] ? gs_change+0x13/0x13
```

**Comment by DeHackEd** [comment]
This appears to have been fixed in bffb68a2b863bc43c42b53ba56d3ee7a4a45ca96 which was also backported (a77cea5f0f1e601d4590130220445afac85547b9) into 0.6.5.8.

**Comment by behlendorf** [comment]
Closing, fix already applied to latest 0.6.5.x release.
